Controllers/WriterController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using IndyBooks.Models;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace IndyBooks.Controllers
+{
+
+    [Route("api/[controller]")]
+    public class WriterController : Controller
+    {
+        //inject the db 
+        private IndyBooksDataContext _db;
+        public WriterController(IndyBooksDataContext db) { _db = db; }
+
+        // GET: api/<controller>
+        [HttpGet]
+        public IActionResult Get()
+        {
+            var writer = _db.Writers.ToList<Writer>();
+            //_db.Add<Writer>(writer);
+
+            return RedirectToAction("Post", writer);
```

Comment:
Here is the issue. The API Controller does NOT use the Post-Redirect-Get process, so each method should finish by either returning JSON or an HTTP message, for example, 
`return Json(writer);`
or 
`return Accepted();`

For the API, each method represents one of the CRUD operations based on the HTTP verb 
Get() - READ
Post() - CREATE
Put() - UPDATE
Delete() - DELETE